orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -50,7 +57,9 @@ def __init__(self):
         self.output_mask = []           # Output corpus indices
         self.doc_webview = None         # WebView for showing content
         self.search_features = []       # two copies are needed since Display allows drag & drop
-        self.display_features = []
+
+        self.search_indices = []
```

Comment:
Currently we have `list_search_indices` and `search_indices` as well as `list_display_indices` and `display_indices`. Why do we need an other set of indices? Wouldn't `search_indices` and `display_indices` suffice?